vaultpress.php

```diff
@@ -2995,7 +2995,7 @@ function get_jetpack_email() {
 		}
 
 		// For version of Jetpack prior to 7.7.
-		if ( ! class_exists( 'Jetpack_IXR_Client' ) ) {
+		if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' )  ) {
```

Comment:
Would it make sense to keep the previous class_exists conditional around as well?

```suggestion
		if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' ) && ! class_exists( 'Jetpack_IXR_Client' ) ) {
```

The prior implementation of the function was

```
/**
 * Loads the Jetpack XML-RPC client
 */
public static function load_xml_rpc_client() {
	require_once ABSPATH . WPINC . '/class-IXR.php';
	require_once JETPACK__PLUGIN_DIR . 'class.jetpack-ixr-client.php';
}
```

so even if called twice it wouldn't double load, but on the odd chance something else was supplying the class from a different file somewhere or other, could cause namespace conflicts?